python/niizarr/_nii2zarr.py

```diff
@@ -200,6 +205,183 @@ def pyramid_labels(x):
+        repeat = space_scale[:1] * (sdim - len(space_scale))
+        space_scale = repeat + space_scale
+    space_scale = space_scale[-sdim:]
+
+    multiscales = [
+        {
+            "version": "0.4",
+            "axes": [
+                {
+                    "name": axis,
+                    "type": axis_to_type[axis],
+                }
+                if axis_to_type[axis] == "channel"
+                else {
+                    "name": axis,
+                    "type": axis_to_type[axis],
+                    "unit": (
+                        space_unit
+                        if axis_to_type[axis] == "space"
+                        else time_unit
+                        if axis_to_type[axis] == "time"
+                        else None
+                    ),
+                }
+                for axis in axes
+            ],
+            "datasets": [],
+            "type": "median window " + "x".join(["2"] * sdim)
+            if not multiscales_type
+            else multiscales_type,
+            "name": name,
+        }
+    ]
+
+    shape0 = shapes[0]
+    for n in range(len(shapes)):
+        shape = shapes[n]
+        multiscales[0]["datasets"].append({})
+        level = multiscales[0]["datasets"][-1]
+        level["path"] = str(n)
+
+        scale = [1] * bdim + [
+            (
+                pyramid_aligns[i] ** n
+                if not isinstance(pyramid_aligns[i], str)
+                else (shape0[bdim + i] / shape[bdim + i])
+                if pyramid_aligns[i][0].lower() == "e"
+                else ((shape0[bdim + i] - 1) / (shape[bdim + i] - 1))
+            )
+            * space_scale[i]
+            if i != no_pool
+            else space_scale[i]
+            for i in range(sdim)
+        ]
+        translation = [0] * bdim + [
+            (
+                pyramid_aligns[i] ** n - 1
+                if not isinstance(pyramid_aligns[i], str)
+                else (shape0[bdim + i] / shape[bdim + i]) - 1
+                if pyramid_aligns[i][0].lower() == "e"
+                else 0
+            )
+            * 0.5
+            * space_scale[i]
+            if i != no_pool
+            else 0
+            for i in range(sdim)
+        ]
+
+        level["coordinateTransformations"] = [
+            {
+                "type": "scale",
+                "scale": scale,
+            },
+            {
+                "type": "translation",
+                "translation": translation,
+            },
+        ]
+
+    scale = [1.0] * ndim
+    if "t" in axes:
+        scale[axes.index("t")] = time_scale
+    multiscales[0]["coordinateTransformations"] = [{"scale": scale, "type": "scale"}]
+
+    multiscales[0]["version"] = ome_zarr_version
+    if ome_zarr_version == "0.4":
+        omz.attrs["multiscales"] = multiscales
+    elif ome_zarr_version == "0.5":
+        omz.attrs["ome"] = {"multiscales": multiscales}
+    else:
+        raise Exception("Unsupported ome version")
```

Comment:
One thing to test is I sort of remember that neuroglancer did not like metadata being under `"ome"` (even if zarr v3, ome 0.5). If that's still the case, we could save the metadata under both `["ome"]["multiscale"]` _and_ `["multiscale"]`.